internal/engine/build_and_deployer_live_update_test.go

```diff
@@ -42,13 +43,36 @@ type testCase struct {
 }
 
 func runTestCase(t *testing.T, f *bdFixture, tCase testCase) {
-	var bs store.BuildStateSet
-	if len(tCase.changedFiles) > 0 {
-		changed := make([]string, len(tCase.changedFiles))
-		for i, file := range tCase.changedFiles {
-			changed[i] = f.WriteFile(file, "blah")
+	bs := store.BuildStateSet{}
```

Comment:
nice! consider putting this in a helper function so it doesn't look like part of the test logic.

Reply:
great idea! done